src/graphrag_toolkit/indexing/load/__init__.py

```
@@ -4,5 +4,7 @@
 from .bedrock_knowledge_base import BedrockKnowledgeBaseExport
 from .file_based_chunks import FileBasedChunks
 from .s3_based_chunks import S3BasedChunks
+from .file_based_docs import FileBasedDocs
```

Ruff (0.8.2)

`.file_based_docs.FileBasedDocs` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)